tests/integration/test_charm.py

```diff
@@ -361,3 +365,107 @@ def generate_s3_config(s3_url: str) -> Dict:
     s3_config["ip_address"] = s3_ip_address
     s3_config["endpoint"] = s3_endpoint
     return s3_config
+
+
+@pytest.mark.asyncio
+@pytest.mark.abort_on_fail
+@pytest.mark.requires_secrets
+@pytest.mark.usefixtures("setup_saml_config")
+async def test_saml_login(  # pylint: disable=too-many-locals
```

Comment:
Should this include a reason why the linting check is disabled?

Reply:
Yes, but I think this line is pretty self-explanatory: there are too many local variables. This test case is quite long but pretty streamlined, so it wouldn't have a lot of readability issues.